docs/tutorial/custom-window-interactions.md

```diff
@@ -0,0 +1,99 @@
+# Custom window interactions
+
+## Custom draggable regions
+
+By default, windows are dragged using the title bar provided by the OS chrome. Apps that remove the default title bar need to use the `app-region` CSS property to define specific areas that can be used to drag the window. Setting `app-region: drag` marks an area as draggable. Setting `app-region: no-drag` excludes an area from dragging. Note that only rectangular shapes are currently supported.
```

Comment:
Do we no longer need the `webkit-` prefix for this CSS parameter anymore?

Reply:
yeah! `app-region` seems to be the main CSS param in Blink's [CSS Properties specification](https://github.com/chromium/chromium/blob/07a6a1c552c161400cecfddf0dd3e393feaaf5fd/third_party/blink/renderer/core/css/css_properties.json5#L6049), tho not sure since when